source/scale/elasticsearch.rst

```diff
@@ -52,7 +52,7 @@ Follow these steps to connect your Elasticsearch server to Mattermost and to gen
 .. note::
 
    - Additional advanced Elasticsearch settings for large deployments can be configured outside the System Console in the ``config.json`` file. Read the `documentation to learn more <https://docs.mattermost.com/administration/config-settings.html#elasticsearch>`__.
-   - If your deployment has a large number of posts (typically in excess of one million but not strictly defined), the reindexing progress percentage may stay at 99% for a long time. The size of the data to be indexed is estimated, and on large databases estimations can become inaccurate. While progress estimates may be inaccurate, and the progress percentage may appear stuck at near completion, indexing will continue behind the scenes until complete.
+   - If your deployment has a large number of posts (typically in excess of one million but not strictly defined), the reindexing progress percentage may stay at 99% for a long time. The size of the data to be indexed is estimated, and on large databases, estimations can become inaccurate. While progress estimates may be inaccurate, and the progress percentage may appear stuck at near completion, indexing will continue behind the scenes until complete.
```

Comment:
```suggestion
   - If your deployment has a large number of posts (typically in excess of one million but not strictly defined), the reindexing progress percentage may stay at 99% for a long time. The size of the data to be indexed is estimated and, on large databases, estimations can become inaccurate. While progress estimates may be inaccurate, and the progress percentage may appear stuck at near completion, indexing will continue behind the scenes until complete.
```

Reply:
I liked the addition of the comma - and then it occurred to me that perhaps the "and" should precede the first comma. What do you think?

Reply:
I thought comma comes before and in cases like this

Reply:
It may be clearer to the reader to split the sentence into two: "The size of the data to be indexed is estimated. On large databases, estimations can become inaccurate.

Reply:
That is a good option too. Splitting the sentence into two @cwarnermm